Gifski/Utilities.swift

```diff
@@ -592,6 +592,54 @@ extension AVAsset {
 		imageGenerator.requestedTimeToleranceBefore = .zero
 		return imageGenerator.image(at: time)
 	}
+
+	func trimBlankFrames() -> AVAsset? {
+		guard let reader = try? AVAssetReader(asset: self) else {
+			return nil
```

Comment:
I followed the existing pattern of returning optionals; but would we want to throw in this extension instead?

Reply:
Yes, the function should be throwing.

Reply:
Fixed in a3a64ed.